astropy/table/table.py

```diff
@@ -740,10 +740,9 @@ def show_in_browser(self,
         jsviewer : bool
             If `True`, prepends some javascript headers so that the table is
             rendered as a https://datatables.net data table.  This allows
-            in-browser searching & sorting.  See `JSViewer
-            <http://www.jsviewer.com/>`_
+            in-browser searching & sorting.
         jskwargs : dict
-            Passed to the `JSViewer`_ init.
+            Passed to the `astropy.table.jsviewer.JSViewer`_ init.
```

Comment:
There isn't a public class called like this. I don't think that's deliberate though. Will you include the fix for that in this PR or should I open a separate one?

Also, the underscore is superfluous at the end of the link.